Source/SwiftLintFramework/Extensions/QueuedPrint.swift

```diff
@@ -34,9 +34,12 @@ public func queuedPrint<T>(object: T) {
 A thread-safe, newline-terminated version of fputs(..., stderr).
 
 - parameter string: String to print.
+- parameter silent: True suppresses output
 */
-public func queuedPrintError(string: String) {
-    dispatch_async(outputQueue) {
-        fputs(string + "\n", stderr)
+public func queuedPrintError(string: String, silent: Bool = false) {
```

Comment:
Instead of a `silent` parameter that just bypasses the function like this, I'd rather just not call the function in the first place. Could you please revert this change?